src/app/common/duck/utils.ts

```diff
@@ -40,13 +41,20 @@ export const handleSelfSignedCertError = (failedUrl: string, dispatch: any) => {
 };
 
 const DNS1123Error = (value) => {
-  return `Invalid character: "${value}". Plan names must be DNS-1123 label compliant, starting and
-    ending with a lowercase alphanumeric character and containing only lowercase alphanumeric
-    characters, "." or "-".`;
+  return `Invalid character: "${value}". Must be DNS-1123 label compliant, starting and ending with
+    a lowercase alphanumeric character and containing only lowercase alphanumeric characters, "."
+    or "-".`;
+};
+
+const nameError = (value) => {
+  return `Invalid name: "${value}". Plan names must start and end with a lowercase alphanumeric
+  character and can contain only lowercase alphanumeric character or one subsequent '-'`;
 };
 
 const testDNS1123 = (value) => DNS1123Validator.test(value);
 
+const testName = (value) => NameValidator.test(value);
```

Comment:
We should have a plan name specific function that first uses the testDNS1123 validation, and if it passes, then additionally applies a validation that ensures no consecutive double dashes are used. Keeps a clean separation of concerns and we can comment specifically the reason for the double dash validation.